From 8d6031adc60d55f69920cd4d6e8d0c6df7b89a1f Mon Sep 17 00:00:00 2001
From: Bijan Tabatabai <bijan311@gmail.com>
Date: Wed, 16 Nov 2022 12:41:11 -0600
Subject: [PATCH 088/179] Revert "Add kernel options for debugging locks"

This reverts commit 8d1a51cc25ae8c1fc15e1259b982dbcf50f86da1.
Turns out it adds some memory overheads that are annoying
---
 kernel_options | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel_options b/kernel_options
index d4c86874f0b1..5808a1ccc412 100644
--- a/kernel_options
+++ b/kernel_options
@@ -57,4 +57,4 @@ CONFIG_VIRTIO_NET=y
 CONFIG_ISO9660_FS=m
 CONFIG_EXT4_FS=y
 
-"setup_wkspc {MACHINE} {USER} --clone_wkspc --git_user BijanT --host_bmks --host_dep --unstable_device_names --resize_root --secret {TOKEN} --spec_2017 /p/multifacet/infrastructure/spec2017.iso" "setup_kernel {MACHINE} {USER} --branch dev --repo github.com/BijanT/linux-file-only-mem --git_user BijanT --secret {TOKEN} --install_perf --fomtierfs +CONFIG_TRANSPARENT_HUGEPAGE +CONFIG_MM_ECON -CONFIG_PAGE_TABLE_ISOLATION -CONFIG_RETPOLINE +CONFIG_GDB_SCRIPTS +CONFIG_FRAME_POINTERS +CONFIG_IKHEADERS +CONFIG_SLAB_FREELIST_RANDOM +CONFIG_SHUFFLE_PAGE_ALLOCATOR +CONFIG_FS_DAX +CONFIG_DAX +CONFIG_BLK_DEV_RAM +CONFIG_FILE_ONLY_MEM +CONFIG_BLK_DEV_PMEM +CONFIG_ND_BLK +CONFIG_BTT +CONFIG_NVDIMM_PFN +CONFIG_NVDIMM_DAX +CONFIG_X86_PMEM_LEGACY -CONFIG_INIT_ON_ALLOC_DEFAULT_ON +CONFIG_DEBUG_KERNEL +CONFIG_LOCK_DEBUGGING_SUPPORT +CONFIG_LOCK_STAT"
+"setup_wkspc {MACHINE} {USER} --clone_wkspc --git_user BijanT --host_bmks --host_dep --unstable_device_names --resize_root --secret {TOKEN} --spec_2017 /p/multifacet/infrastructure/spec2017.iso" "setup_kernel {MACHINE} {USER} --branch dev --repo github.com/BijanT/linux-file-only-mem --git_user BijanT --secret {TOKEN} --install_perf --fomtierfs +CONFIG_TRANSPARENT_HUGEPAGE +CONFIG_MM_ECON -CONFIG_PAGE_TABLE_ISOLATION -CONFIG_RETPOLINE +CONFIG_GDB_SCRIPTS +CONFIG_FRAME_POINTERS +CONFIG_IKHEADERS +CONFIG_SLAB_FREELIST_RANDOM +CONFIG_SHUFFLE_PAGE_ALLOCATOR +CONFIG_FS_DAX +CONFIG_DAX +CONFIG_BLK_DEV_RAM +CONFIG_FILE_ONLY_MEM +CONFIG_BLK_DEV_PMEM +CONFIG_ND_BLK +CONFIG_BTT +CONFIG_NVDIMM_PFN +CONFIG_NVDIMM_DAX +CONFIG_X86_PMEM_LEGACY -CONFIG_INIT_ON_ALLOC_DEFAULT_ON"
-- 
2.49.0

